plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/testdsl/testAnnotations.kt

```diff
@@ -12,10 +12,10 @@ import kotlin.streams.asStream
 
 object TestVersions {
     const val minSupportedGradleVersion = KtlintBasePlugin.LOWEST_SUPPORTED_GRADLE_VERSION
-    const val maxSupportedGradleVersion = "7.1.1"
+    const val maxSupportedGradleVersion = "7.4.2"
```

Comment:
We like keeping the same version range support as the Google android plugin in general. Is this still kept with this change?

Reply:
Support as high as possible. But make sure we're not loosing versions still supported by the AGP